apps/web/app/(app)/gardens/[chain]/[garden]/pool/[poolId]/proposals/[proposalId]/page.tsx

```diff
@@ -1,16 +1,19 @@
+"use client";
+
```

Comment:
now its changed to client, all the contracts calls its made from client, maybe in the future take from subgraph some info.

Reply:
You mean the contract read ? Is it a problem that its made though client ? Dont think we should rely too much on subgraph when not necessary becaus cost more

Reply:
cost its not easy, performance its, but yeah i think even that call contracts can be improved with new function in contract to do one call.

Reply:
Yeah but to be honest right now fetching the chain ios way more faster to be able to requery, we don't need the indexing delay